Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -253,10 +254,35 @@ public class Snapshotter {
 }
 
 extension Snapshotter: ObservableProtocol {
+    /// Subscribes an Observer to a provided list of event types.
+    ///
+    /// A type conforming to `ObservableProtocol` will hold a strong reference
+    /// to an Observer instance, therefore, in order to stop receiving notifications,
+    /// the caller must call `unsubscribe` with the instance used for an initial
+    /// subscription.
+    ///
+    /// - Parameters:
+    ///   - observer: An `Observer`
+    ///   - events: Array of event types to be subscribed to
+    ///
+    /// - Note:
+    ///     Prefer `MapboxMap.on()` and `Snapshotter.on()` to using these
+    ///     lower-level APIs
     public func subscribe(_ observer: Observer, events: [String]) {
         mapSnapshotter.subscribe(for: observer, events: events)
     }
 
+    /// Unsubscribes an Observer from a provided list of event types.
+    ///
+    /// A type conforming to `ObservableProtocol` will hold a strong reference
+    /// to an Observer instance, therefore, in order to stop receiving notifications,
+    /// the caller must call `unsubscribe` with the instance used for an initial
+    /// subscription.
+    ///
+    /// - Parameters:
+    ///   - observer: An `Observer`
```

Comment:
This doesn't feel like a very useful comment. What is it observing?

Reply:
I'd like to ticket this. I merely relocated the existing documentation.

Reply:
Nevermind, I made some updates to these comments